available_dishes.csv

```diff
@@ -0,0 +1,8 @@
+Tikka Masala,11.00,1
+Cheeseburger,11.50,2
+Hamburger,10.50,3
+Margarita Pizza,10.00,4
+Pepperoni Pizza,11.50,5
+Veggie Pizza,11.00,6
+Messy Fries,9.50,7
+Chips,4.50,8
```

Comment:
This approach feels a little overcomplicated - writing to / reading a CSV is perhaps a little beyond the scope of this challenge. It's encouraged to keep things as a simple as possible in your implementations, and you could define each of these dishes as an instance of Dish, for example, stored in an array in a 'Menu' object.